<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="VolumeEditorWidget" name="viewer"/>
      <widget class="LayerWidget" name="layerWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QToolButton" name="UpButton">
        <property name="toolTip">
         <string>move layer up</string>
        </property>
        <property name="text">
         <string>Up</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/go-up.png</normaloff>:/icons/icons/go-up.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="DownButton">
        <property name="toolTip">
         <string>move layer down</string>
        </property>
        <property name="text">
         <string>Down</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/go-down.png</normaloff>:/icons/icons/go-down.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="DeleteButton">
        <property name="toolTip">
         <string>delete layer</string>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/list-remove.png</normaloff>:/icons/icons/list-remove.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="AddLayOutButton">
        <property name="toolTip">
         <string>add new layer</string>
        </property>
        <property name="text">
         <string>Add Layout</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/list-add.png</normaloff>:/icons/icons/list-add.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/process-stop.png</normaloff>:/icons/icons/process-stop.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionAllZoomToFit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/view-fullscreen.png</normaloff>:/icons/icons/view-fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Zoom to Fit</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionToggleHUDs">
   <property name="text">
    <string>Toggle HUDs</string>
   </property>
  </action>
  <action name="actionCenterImages">
   <property name="text">
    <string>Center Images</string>
   </property>
  </action>
  <action name="actionOnly_for_selected_view">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Only for selected view</string>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionSelectedZoomToFit">
   <property name="text">
    <string>Fit Image</string>
   </property>
  </action>
  <action name="actionToggleHUD">
   <property name="text">
    <string>Toggle HUD</string>
   </property>
  </action>
  <action name="actionCenterImage">
   <property name="text">
    <string>Center Image</string>
   </property>
  </action>
  <action name="actionResetZoom">
   <property name="text">
    <string>Reset Zoom</string>
   </property>
  </action>
  <action name="actionRubberBandZoom">
   <property name="text">
    <string>Rubber Band Zoom</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LayerWidget</class>
   <extends>QListView</extends>
   <header>volumina.widgets.layerwidget</header>
  </customwidget>
  <customwidget>
   <class>VolumeEditorWidget</class>
   <extends>QWidget</extends>
   <header>volumina.volumeEditorWidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
